collect_app/src/main/java/org/odk/collect/android/spatial/MapHelper.java

```diff
@@ -320,4 +318,8 @@ private boolean isFileFormatSupported(File file) {
         }
         return result;
     }
+
+    public int getSelectedLayer() {
```

Comment:
Since `selectedLayer` can be null, this method might crash.  Shouldn't it return an `Integer` instead?

Also, it would be good to add a docstring for this method to explain what this integer is (e.g. "Returns the integer index of (whatever) in (whatever), or null when (whatever).")